Packs/Trello/Integrations/Trello/Trello.yml

```diff
@@ -0,0 +1,520 @@
+      - |
+        enablePlugin
+      - |
+        enablePowerUp
+      - |
+        makeAdminOfBoard
+      - |
+        makeAdminOfOrganization
+      - |
+        makeNormalMemberOfBoard
+      - |
+        makeNormalMemberOfOrganization
+      - |
+        makeObserverOfBoard
+      - |
+        memberJoinedTrello
+      - |
+        moveCardFromBoard
+      - |
+        moveCardToBoard
+      - |
+        moveListFromBoard
+      - |
+        moveListToBoard
+      - |
+        removeAdminFromBoard (Deprecated in favor of makeNormalMemberOfBoard) removeAdminFromOrganization (Deprecated in favor of - makeNormalMemberOfOrganization)
+      - |
+        removeChecklistFromCard
+      - |
+        removeFromEnterprisePluginWhitelist
+      - |
+        removeFromOrganizationBoard
+      - |
+        removeLabelFromCard
+      - |
+        removeMemberFromBoard
+      - |
+        removeMemberFromCard
+      - |
+        removeMemberFromOrganization
+      - |
+        removeOrganizationFromEnterprise
+      - |
+        unconfirmedBoardInvitation
+      - |
+        unconfirmedOrganizationInvitation
+      - |
+        updateBoard
+      - |
+        updateCard
+      - |
+        updateCheckItem
+      - |
+        updateCheckItemStateOnCard
+      - |
+        updateChecklist
+      - |
+        updateLabel
+      - |
+        updateList
+      - |
+        updateMember
+      - updateOrganization
+      - voteOnCard
+      required: false
+      secret: false
+    - default: false
+      description: First action time
+      isArray: false
+      name: since
+      required: false
+      secret: false
+    - default: false
+      description: Last action time
+      isArray: false
+      name: before
+      required: false
+      secret: false
+    deprecated: false
+    description: List all actions on a board, such as card updates, additions, and
+      deletes.
+    execution: false
+    name: list-actions
+    outputs:
+    - contextPath: Trello.Actions.Id
+      description: Action ID
+      type: Unknown
+    - contextPath: Trello.Actions.Type
+      description: Type of action
+      type: Unknown
+    - contextPath: Trello.Actions.Date
+      description: Date of action
+      type: Date
+    - contextPath: Trello.Actions.ListId
+      description: List ID - Null if not card or list action
+      type: Unknown
+    - contextPath: Trello.Actions.CardId
+      description: Card ID - Null if not card action
+      type: Unknown
+    - contextPath: Trello.Actions.BoardId
+      description: Board ID
+      type: Unknown
+  - arguments:
+    - default: false
+      description: ID to get
+      isArray: false
+      name: id
+      required: false
+      secret: false
+    - default: false
+      description: Last update
+      isArray: false
+      name: lastUpdate
+      required: false
+      secret: false
+    deprecated: false
+    description: Used by mirroring
+    execution: false
+    name: get-remote-data
```

Comment:
did you plan to implement incoming mirroring? 
if so, some stuff are missing, such as the `isremotesyncin: true` is missing from the YML and the handling in the code and the mapper

Reply:
As discussed, this command should be removed from the YML file.

Reply:
Removed